scripts/docker-integration-tests/aggregator_tls/test.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+
+set -xe
+
+source "$M3_PATH"/scripts/docker-integration-tests/common.sh
```

Comment:
@roman-mazhut just for future info--there are Go integration tests now which are preferred over the shell ones.